salvo/salvo.py

```diff
@@ -3,10 +3,10 @@
 import argparse
 import logging
 import os
-import site
 import sys
 
 from src.lib.job_control_loader import load_control_doc
+from src.lib.run_benchmark import BenchmarkRunner
```

Comment:
Can we change the import, so that it imports the module rather than the class?

https://github.com/google/styleguide/blob/gh-pages/pyguide.md#22-imports

Reply:
Certainly.  This instance flew under the radar.